cafebabel/tests/test_articles_archives.py

```diff
@@ -6,7 +6,7 @@
 
 
 def test_archive_is_redirect_to_article_with_lang(client, published_article):
-    url = 'http://cafebabel.test/lifestyle/article/ancien-article.html'
+    url = 'http://cafebabel.test/lifestyle/article/old-article.html'
```

Comment:
That was an article in fr, hence the "ancien" :p
(showing off cause my French is so good)